integrations.yml

```diff
@@ -94,3 +94,9 @@
   website: "https://automate.io/integration/basecamp-3"
   icon: "automate.io.png"
   category: 3	
+-
+  name: "OAuth.io"
+  description: "OAuth that just works ! Ease the OAuth integration of Basecamp API v3 and interact with its API in your web and mobile applications."
```

Comment:
I came to this repository after this discussion on Twitter: https://twitter.com/basecamp/status/804070609923641344

It's a service + SDKs (Javascript / iOS / Android / Phonegap / PHP / Node.js / Go) which give:
- Easy authorization: `OAuth.popup('basecamp')`
- Easy way to call your API - automatic tokens injection in API calls (see screenshot bellow)
- Unified user information - `me()` always send the same structure independently of the OAuth provider (see screenshot bellow)
- User management - Signin/Signup with Basecamp (Authentication)

![image](https://cloud.githubusercontent.com/assets/1288646/21000443/4588f19a-bcf1-11e6-8843-64cc85ca0d53.png)

It could enhance the developer experience when developers want to integrate Basecamp in their apps. It's maybe the wrong place to post it (the best place I think would be in your API documentation itself) but as someone on Twitter linked me to this, I thought it would be a good starting point :)

The core is opensource (and can be installed for free anywhere) and the integration has been implemented here: https://github.com/oauth-io/oauthd/tree/master/providers/basecamp - If someone from Basecamp wants to update something (logo / description / screenshot / integration itself), PR are welcome !